pkg/generators/markers_test.go

```diff
@@ -325,6 +332,58 @@ func TestParseCommentTags(t *testing.T) {
 				},
 			},
 		},
+		{
+			t:    types.Float64,
+			name: "raw string rule",
+			comments: []string{
+				`+k8s:validation:cel[0]:rule> raw string rule`,
```

Comment:
question: has this design with `>` representing multiline been flushed out somewhere or is this still experimental?

Reply:
It's the result of some offline discussions I've had with a few folks. The KEP makes reference to multiline support being desired, but no specific syntax was perscibed

Reply:
> The KEP makes reference to multiline support being desired, but no specific syntax was perscibed

I think this is OK.  It's developer facing. It's not part of our forever APIs. We could change it if we really wanted to.  Number of call sites in k/k will probably be in the 100s..

Reply:
sgtm